app/helpers/measures_helper.rb

```diff
@@ -21,4 +23,14 @@ def legal_act_regulation_url_link_for(measure)
 
     sanitize(link, attributes: %w[href target role rel class title])
   end
+
+  def check_how_to_export_goods_link(commodity_code:, country_code:, country_name:, eu_member:)
+    if eu_member
+      link_to("Check how to export #{commodity_code} to #{country_name} (link opens in new tab)",
+              "#{CHECK_DUTIES_SERVICE_URL}/summary?d=#{country_code}&ds=gtp&tab=tree&pc=#{commodity_code}",
```

Comment:
I think it makes sense for service config to go in `TradeTariffFrontend.check_duties_service_url`

Reply:
```ruby
module TradeTariffFrontend
  def self.check_duties_service_url
    ENV.fetch('CHECK_DUTIES_SERVICE_URL', 'https://www.check-duties-customs-exporting-goods.service.gov.uk')
  end
end
```